### PR TITLE
Add Row Support

### PR DESCRIPTION
**Pull request by banoop**
Created Add Row function which consolidates attribute values of object into an INSERT INTO, VALUES sql command. Add row is called on object creation either when table exists for object or not. Object-hash used to give hash values for objects. Object-hash installed using npm.

### CODE REVIEW

File: DiscreetORM.ts [hunk 1]
```diff
@@ -104,12 +105,25 @@ export class StoredClass implements ObjectListener<any>{
         this.discreet_sql_io.writeSQL(escaped_command);
         this.discreet_sql_io.writeNewTable(table_name);
     }
-
+    addRow(obj: any) {
+                let table_name = obj.constructor.name;
+                let command = "INSERT INTO " + table_name + "\n" + "VALUES (";
```
Comment (haramkim-1) [inline DiscreetORM.ts]: command construction should use sqlstring to create escaped SQL string

Comment (T40TDS) [reply DiscreetORM.ts]: +1

**Review by T40TDS — CHANGES_REQUESTED** [review]
Please use the sqlstring library so we don't have injection risk

File: DiscreetORM.ts [hunk 2]
```diff
@@ -104,12 +105,25 @@ export class StoredClass implements ObjectListener<any>{
...
+                var hash = require('object-hash');
```
Comment (T40TDS) [inline DiscreetORM.ts]: This should be with the require statements at the top of the file

**Review by T40TDS — CHANGES_REQUESTED** [review]

**Review by T40TDS — APPROVED** [review]
If it passes inspection test, like it outputs something that's reasonable to output/commands.sql, land it

**Pull request merged**